src/_includes/blocks/footer.njk

```diff
@@ -1,11 +1,81 @@
- <footer class="border-t text-xs pt-4 mt-4">
+ <div class="text-xs mt-4 mx-4 mb-0">
+ The Globe Church is all about Jesus, made up of all sorts of people, involved in the greatest mission... 
+ </div>
+ 
+ <footer class="border-t text-xs pt-4 mt-3 mx-4">
+
+<!--
   <div>
     {{ meta.registeredAddress}}
   </div>
+-->
 
-  <div>
-    &copy; Copyright 2015 – {{ data.currentYear }} The Globe Church &middot;
+ <nav>
+  <div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-6 lg:grid-cols-6 gap-5">
+      <div>
+        <ul class="list-none">
+          <li><a class="text-sm text-black" href="/about">About</a></li>
```

Comment:
For the moment can we leave these as the default text colour. Still undecided around if the footer should be pure black text or not